Bindings/bindings.json

```diff
@@ -1359,6 +1359,57 @@
                     "help": "Choose platform to send native notifcations. Select Template to send template notification."
                 }
             ]
+        },
+        {
+            "type": "sendGrid",
+            "displayName": "SendGrid",
+            "direction": "out",
+            "settings": [
+                {
+                    "name": "to",
+                    "value": "string",
+                    "defaultValue": "",
+                    "required": false,
+                    "label": "To address",
+                    "help": "To address that the message should be sent to.",
+                    "validators": [
+                        {
+                            "expression": "TODO",
+                            "errorText": "Please enter a valid email address."
+                        }
+                    ]
+                },
+                {
+                    "name": "from",
+                    "value": "string",
+                    "defaultValue": "",
+                    "required": false,
+                    "label": "From address",
+                    "help": "The address that the message is sent from.",
+                    "validators": [
+                        {
+                            "expression": "TODO",
+                            "errorText": "Please enter a valid email address."
+                        }
+                    ]
+                },
+                {
+                    "name": "subject",
+                    "value": "string",
+                    "defaultValue": "",
+                    "required": false,
+                    "label": "Message Subject",
+                    "help": "The Subject line to use for the message."
```

Comment:
Email Text and Subject line are pretty open ended. I don't think we need a regex here.